### PR TITLE
[HDRP] Remove MotionVector of UI for VertexAnimation and small fixes

### PR DESCRIPTION
**Pull request by RSlysz**
### Purpose of this PR
- Fix FB https://fogbugz.unity3d.com/f/cases/1133546/
- Add missing AxF SetupKeywordsAndPass in HDEditorUtils.ResetMaterialKeywords
Comment HDEditorUtils and clean visibility
- Fix gitattribut forcing line ending for *.asset that conflict with LightingData.asset in test project containing binary content. It cause to not be able to remove change on them as they got changed back right after the checkout. This prevent rebasing branch.
- Renamed Base UV Mapping in Emission to Emission UV Mapping

---
### Release Notes

---
### Testing status
**Katana Tests**: 

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] Checked new UI names with UX convention
- Other: 

**Automated Tests**: 

---
### Overall Product Risks
**Technical Risk**: Medium

**Halo Effect**: Low

---
### Comments to reviewers
@fredpointzero mainly for visibility change in HDEditorUtils. Is there any reason we had everything public ?

Many file changed but only to remove the unused code on wind. Except for HDEditorUtils and .gitattributes.

@JulienIgnace-Unity Cannot managed to rebase in master branch as I got several conflict. It seams that the common git ancestor is a long time ago. It should be missing a merge master -> HDRP/staging or HDRP/staging -> master

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Material/Lit/BaseLitUI.cs [hunk 1]
```diff
@@ -149,20 +149,6 @@ public enum HeightmapParametrization
         protected MaterialProperty invPrimScale = null;
         protected const string kInvPrimScale = "_InvPrimScale";
 
-        // Wind
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/Material/Lit/BaseLitUI.cs]: be careful, we can't remove it before Fontainbleau demo move wind to shader graph

File: com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitUI.cs [hunk 2]
```diff
@@ -776,15 +769,7 @@ static public void SetupBaseUnlitMaterialPass(Material material)
...
-            if (material.HasProperty(kEnableMotionVectorForVertexAnimation))
-            {
-                material.SetShaderPassEnabled(HDShaderPassNames.s_MotionVectorsStr, material.GetFloat(kEnableMotionVectorForVertexAnimation) > 0.0f);
-            }
-            else
-            {
-                // In case we have an HDRP material that inherits from this UI but does not have an _EnableMotionVectorForVertexAnimation property, we need to set it to false (default behavior)
-                material.SetShaderPassEnabled(HDShaderPassNames.s_MotionVectorsStr, false);
-            }
+            material.SetShaderPassEnabled(HDShaderPassNames.s_MotionVectorsStr, false);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitUI.cs]: Please add a comment here:
"We don't have any vertex animation for lit/unlit vector, so we setup motion vector pass to false. Remind that
in HDRP this doesn't disable motion vector, it just mean that the material don't do any vertex deformation but
we can still have skinning/morph target"

**Comment by RSlysz** [comment]
If there is an issue with 2019.1 and fontainebleau, can we at least partly backport the side fixes?
Side fixes:
- f694f46
- 5ff6a5e
- 4780f5e
- 133fa4b

**Comment by sebastienlagarde** [comment]
Note: Let's check with Laurent what is the state of wind with Fontainbleau tomorrow, and also @alelievr have a big PR that could conflict with this one

**Comment by alelievr** [comment]
Yes, i'll probably need to manually merge all the UI files

**Comment by sebastienlagarde** [comment]
Hey, we will move the change of this PR into this one: https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/3692

**Pull request closed without merging**